xbmc/interfaces/builtins/SkinBuiltins.cpp

```diff
@@ -112,14 +112,18 @@ static int SelectBool(const std::vector<std::string>& params)
 
   CGUIDialogSelect* pDlgSelect = (CGUIDialogSelect*)g_windowManager.GetWindow(WINDOW_DIALOG_SELECT);
   pDlgSelect->Reset();
-  pDlgSelect->SetHeading(CVariant{g_localizeStrings.Get(424)});
+  int iHeader = atoi(params[0].c_str());
```

Comment:
there's already a safeguard in place for that, the number of mandatory arguments are defined here:
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/builtins/SkinBuiltins.cpp#L584

`ERROR: Keymapping error: no such action 'skin.selectbool()' defined`
`DEBUG: ExecuteXBMCAction: Tried translating, but failed to understand skin.selectbool()`

i will bump the value to two though.